wordpress-mobile/WordPress-Android
# Stats revamp v2 most popular time card

This PR updates the existing Most Popular Time card to new design. 


Fixes #16397 

| Before | New Design | Dark Theme |
|---|---|---|
|![Screenshot_20220427_221059](https://user-images.githubusercontent.com/990349/165516688-395af98d-b330-443b-b085-d3ebb494fe70.png)|![Screenshot_20220427_220824](https://user-images.githubusercontent.com/990349/165516736-01b9bc41-3ef8-4471-bcc5-b84430b0773f.png)|![Screenshot_20220427_221413](https://user-images.githubusercontent.com/990349/165516797-0bb72203-dc91-4139-b7a2-4210234e336d.png)|


**Known issues**

- This currently doesn't cover the 'no data' case – this will come in a future PR:
- Styles will be updated in a separate PR

**To test**

Setup:

- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
- Select Debug Settings
- Find StatsRevampV2FeatureConfig under Features in development enable it and restart app

Test 1:

- Launch app
- Go to stats either using quick links or menu
- Ensure that it opens in Insights tab otherwise switch to Insights tab
- Verify that the new Most Popular Time card is visible and matches the new designs above
- Ensure that 🔥 emoji is prefixed to the title, over flow menu is hidden and % views label below day and time
- If the Most Popular Time card is not present.  Go to stats management (either through ⚙️ menu at the top right hand corner or Add new stats card at the bottom) and add the card and save then return to Insights.

Test 2:

- Turn on the dark theme on your device
- Check that it card is displayed as in third image above

Test 3:

- Disable StatsRevampV2FeatureConfig and restart app
- Return to Stats and make sure Most Popular Time card is displayed as in first image above as before.



## Regression Notes
1. Potential unintended areas of impact
Existing Most Popular Time card will be update to new design.  No change in data

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
